File: api-docs/packages/wfp-hdx-lka-food-docs/python/smoke.py
"""Quick client smoke — polite delay on."""

from __future__ import annotations

import json

from wfp_hdx_lka_food_docs import WfpHdxLkaFoodDocsClient


def json_preview(data: object, limit: int = 200) -> str:
    try:
        return json.dumps(data, ensure_ascii=False)[:limit]
    except TypeError:
        return str(data)[:limit]


with WfpHdxLkaFoodDocsClient(default_delay_seconds=1.0) as client:
    print("smoke", client.slug, "->", "wfp_food_prices_lka_csv")
    data = client.wfp_food_prices_lka_csv()
    preview = data[:200] if isinstance(data, str) else json_preview(data)
    print("ok", preview)

File: api-docs/packages/boc-rates-docs/javascript/examples/smoke.mjs
import { BocRatesDocsClient } from "../client.mjs";

const client = new BocRatesDocsClient({ defaultDelayMs: 1000 });
console.log("smoke", BocRatesDocsClient.slug, "->", "ratesTariffHtml");
const data = await client.ratesTariffHtml();
const preview = typeof data === "string" ? data.slice(0, 200) : JSON.stringify(data)?.slice(0, 200);
console.log("ok", preview);
